roles/openshift_facts/library/openshift_facts.py

```diff
@@ -56,6 +56,13 @@ def migrate_docker_facts(facts):
     if 'node' in facts and 'portal_net' in facts['node']:
         facts['docker']['hosted_registry_insecure'] = True
         facts['docker']['hosted_registry_network'] = facts['node'].pop('portal_net')
+
+    # log_options was originally meant to be a comma separated string, but
+    # we now prefer an actual list, with backward compatability:
+    if 'log_options' in facts['docker'] and \
+            isinstance(facts['docker']['log_options'], basestring):
+        facts['docker']['log_options'] = facts['docker']['log_options'].split(",")
```

Comment:
Looks good for the migration use case...  You'll need to add something similar to `init_local_facts` to handle the conversion from string to array there as well to avoid persisting a string on the first time through the role.